File: w7l2.cpp
//checking if a file exists in c++
// #include<fstream>
// #include<iostream>
// using namespace std;
// int main(){
//     ifstream file("yes.txt");
//     if (file){
//         cout<<"FILE EXISTS.\n";
//     }
//     else{
//         cout<<"FILE DOES NOT EXIST.";
//     }
//     return 0;
// }


//Writing in c++
#include <fstream>
using namespace std;
int main()
{
    ofstream myfile("example.txt");
    if (myfile.is_open()){
        myfile<<"LINE 1\n";
        myfile<<"LINE 2\n";
        myfile<<"LINE 3\n";
        myfile.close();
    }
    return 0;
}

File: w7l2.java
// import java.io.BufferedReader;
// import java.io.FileReader;
// import java.io.IOException;

// public class w7l2 {
//     public static void main(String[] args) {
//         try{
//             BufferedReader reader=new BufferedReader(new FileReader("example.txt"));
//             String Line;
//             while ((Line=reader.readLine())!=null){
//                 System.out.println(Line);
//             }
//             reader.close();
//         }
//         catch(IOException e){
//             e.printStackTrace();
//         }
//     }
    
// } 


// checking if a file exists in java
import java.io.File;
public class w7l2{
    public static void main(String[] args) {
        File file=new File("example.txt");
        if(file.exists()){
            System.out.println("FILE EXISTS\n");
        }
        else{
            System.out.println("File does not exist\n");
        } 
    }
}